internal/selectors/selectors.go

```diff
@@ -158,19 +158,75 @@ func GetNoneSelector(inst *unstructured.Unstructured) (bool, error) {
 	return noneSelector, nil
 }
 
-// cmExclusions are known (istio and kube-root) CA configmap which are excluded from propagation
-var cmExclusions = []string{"istio-ca-root-cert", "kube-root-ca.crt"}
+type ExclusionByNameSpec struct {
+	// Group of the resource defined below. This is used to unambiguously identify the resource. It
+	// may be omitted for core resources (e.g. "secrets").
+	Group string
+
+	// Kind of the resource
+	Kind string
+
+	// Name of the resoruce - can be used to filter by name of the object
+	Name string
+}
+
+type ExclusionByLabelsSpec struct {
+	// Group of the resource defined below. This is used to unambiguously identify the resource. It
+	// may be omitted for core resources (e.g. "secrets").
+	Group string
+
+	// Kind of the resource
+	Kind string
+
+	// Labels - can be used to filter by label; both key and value are used to match the label, i.e.
+	// value of the label has also to match. If multiple labels are set, ALL of them have to match!
+	Labels map[string]string
+}
+
+// ExclusionsByName are known (istio and kube-root) CA configmap which are excluded from propagation
+var ExclusionsByName = []ExclusionByNameSpec{
+	{Kind: "ConfigMap", Name: "istio-ca-root-cert"},
+	{Kind: "ConfigMap", Name: "kube-root-ca.crt"},
+}
+
+var ExclusionByLabels = []ExclusionByLabelsSpec{
+	{Kind: "ServiceAccount", Labels: map[string]string{"cattle.io/creator": "norman"}},
+	{Group: "rbac.authorization.k8s.io", Kind: "RoleBinding", Labels: map[string]string{"cattle.io/creator": "norman"}},
+	{Group: "rbac.authorization.k8s.io", Kind: "Role", Labels: map[string]string{"cattle.io/creator": "norman"}},
+	{Group: "networking.k8s.io", Kind: "NetworkPolicy", Labels: map[string]string{"cattle.io/creator": "norman"}},
```

Comment:
Is there ever a case where we _wouldn't_ want to exclude something with this label? If so, maybe take out `Group` and `Kind` for now, and just test the label? I see you've added a test for this (which is great) but it might be worth adding a comment why only these _specific_ Kinds shouldn't be propagated.

Also, if you could link to a site that defines what `cattle.io/creator:norman` is, that'd be appreciated.